backend/package.json

```diff
@@ -8,7 +8,9 @@
     "lint": "eslint .",
     "lintfix": "eslint --fix .",
     "start": "babel-node server",
-    "test": "mocha tests --recursive --require babel-polyfill --require @babel/register"
+    "mockTest": "bash test.sh",
+    "dockerTest": "mocha tests --recursive --require babel-polyfill --require @babel/register",
+    "hostedTest": "mocha tests --recursive --require babel-polyfill --require @babel/register"
```

Comment:
DockerTest and HostedTest have the same script

Reply:
DockerTest is what docker runs, I just put hostedTest for users to run the tests against a pre-setup hosted database.
Can probably remove one, just thought it would be more clear have two.